deployment/ansible/roles/raster-foundry.docker/templates/daemon.json.j2

```diff
@@ -0,0 +1,15 @@
+{
+    "userns-remap": "{{ '' if docker_user == 'vagrant' else 'jenkins' }}",
```

Comment:
The default is for `userns-remap` to be an empty string. I believe that https://github.com/raster-foundry/raster-foundry/pull/4934 recently broke newly provisioned development environments, so this attempts to fix it, in addition to adding configuration for `default-ulimits`.